src/components/__tests__/Loader.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { render } from '@testing-library/react';
+import Loader from '../Loader/Loader';
+import { describe, it, expect } from 'vitest';
+
+describe('Loader', () => {
+  it('renders loader spinner', () => {
+    const { container } = render(<Loader />);
+    const spinner = container.querySelector('.animate-spin');
```

Comment:
it's better not to use querySelectors in tests if possible, try to search another option